bin/compile

```diff
@@ -133,9 +133,9 @@ list_node_config | output "$LOG_FILE"
 create_build_env
 
 ### Configure package manager cache directories
-[ ! "$YARN_CACHE_FOLDER" ] && YARN_CACHE_FOLDER=$(mktemp -d -t yarncache.XXXXX)
+# [ ! "$YARN_CACHE_FOLDER" ] && YARN_CACHE_FOLDER=$(mktemp -d -t yarncache.XXXXX)
```

Comment:
Yarn 2 cache is now included in the repo (at least for the zero-installs setup) so I believe the $YARN_CACHE_FOLDER env var is not needed anymore